webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -39,11 +39,16 @@ class SubscriptionsPage extends Component {
 
   componentDidUpdate(prevProps) {
     const {
-      organization, task, handleTask,
+      organization, task, handleStartTask, handleFinishedTask, isTaskPending, isPollingTask,
     } = this.props;
-
     if (task) {
-      handleTask(task);
+      if (isPollingTask) {
```

Comment:
Let's get some test coverage here too :)

Reply:
I haven't actually tested these changes yet but I think there's an issue here. Based on the way the code reads, I think we'll only update the progress bar modal when we start polling a task and when that task completes so we'll miss the 'progress' aspect of the modal.